blockchain/chain/src/store/base_fee.rs

```diff
@@ -13,7 +13,7 @@ use std::collections::HashSet;
 /// Used in calculating the base fee change.
 pub const BLOCK_GAS_TARGET: i64 = BLOCK_GAS_LIMIT / 2;
 
-/// Limits gas base fee change to 12.5% of the change.
+/// Limits gas base fee change to 12.5 percent of the change.
```

Comment:
hunspell doesn't understand `%`?

Reply:
No and didn't find a way to make it work.

Reply:
Maybe using similar `hunspell.quirks` settings? [like here](https://gitlab.parity.io/parity/mirrors/polkadot/-/blob/master/scripts/ci/gitlab/spellcheck.toml)